src/bp/ui-studio/src/web/reducers/selectors.ts

```diff
@@ -48,20 +48,18 @@ export const getCurrentFlowSubFlows = createSelector(
   [_getFlowsByName, _getCurrentFlow],
   (flowsByName, currentFlow: string): ParsedFlowDefinition[] => {
     return Object.keys(flowsByName)
-      .filter(name => flowsByName[name].parent === currentFlow && !isSkillFlow(name))
+      .filter(name => flowsByName[name].type === 'reusable' && !isSkillFlow(name))
       .map(name => parseFlowName(name, true))
   }
 )
```

Comment:
Hmm, this method doesn't do what its name implies. I would expect this method to return the list of reusable workflows that the current flow is using, and you are no longer using the currentFlow, therefore it would return all reusable.  

Either fix the name or change the logic (and remove the unused var if you don't need it)